test/ComponentTests/PlatformServices.Desktop.Component.Tests/App.config

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
please can you explain, why did you delete this file?

Reply:
I did some cleaning, this file is actually no longer relevant (was already the case with main).